module/userCourses/helper.js

```
@@ -0,0 +1,211 @@
+					...(userCoursesData.status.toLowerCase() === CONSTANTS.common.SUBMISSION_STATUS_COMPLETED && {
+						completedAt: new Date(),
+					}),
+				}
+			}
+
+			let updateObject = { $set: updateCourseData }
+
+			let updateCourse = await userCoursesQueries.updateMany(coursesfilter, updateObject, { new: true })
+			if (!updateCourse || updateCourse.nModified === 0) {
+				return {
+					status: HTTP_STATUS_CODE.bad_request.status,
```

ESLint

[error] 'HTTP_STATUS_CODE' is not defined.

(no-undef)